ballsdex/core/bot.py

```diff
@@ -327,7 +327,7 @@ async def on_ready(self):
         else:
             log.info("No command to sync.")
 
-        if "admin" in PACKAGES:
+        if "admin" in settings.packages:
```

Comment:
Doesn't sync without the full path
```suggestion
        if "ballsdex.packages.admin" in settings.packages:
```